plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -11,6 +11,10 @@ $font-sf-pro-display: "SF Pro Text", -apple-system, BlinkMacSystemFont, "Segoe U
 	}
 }
 
+.woocommerce-woopayments-ready-to-test-modal{
+	font-family: $font-sf-pro-display;
```

Comment:
not sure if it's the best idea to put it here

Reply:
What do you mean?

Reply:
it's already out-of date, closing